### ISSUE TITLE
Change the way rolling a seven works

### ISSUE THREAD

**Issue by Tacitor** [Tacitor/ICS4U-Settlers-Of-Catan#71]
The thief mechanics are still a little odd. What happened was this:
1) Red rolls a 7. Nothing happens yet, need to click to next turn.
2) It's Blue's turn now, need to give up cards
3) Red's turn again, need to decide which cards to give up.
5) Back to Red's turn but they are not allowed to take any actions.
6) Now Blue rolls a new number and takes their turn.

Typically when rolling a 7 this is what would happen:
1) Red rolls a 7. If they have >7 cards they give up half.
2) Next Blue would give up half if they have >7
3) Now Red gets to play as normal. When finished they click next turn
4) Blue rolls a new number and gets to play

In summary it seems like the player rolling a 7 does not get to play their turn (i.e building).
